app/models/favorite.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Favorite < ApplicationRecord
+  validates :user_id, presence: true
+  validates :tweet_id, presence: true
```

Comment:
以下のように書いた方が良いかなと。
（belongs_to書くだけでバリデーション実行されるけども。）
Ref: https://github.com/Progaku-copy/progaku-archive/pull/21#discussion_r1377707231

```suggestion
  validates :user, presence: true
  validates :tweet, presence: true
```

Reply:
カラム名を変更してマイグレーションやりなおすってこと？

Reply:
下記コミットで修正しました
https://github.com/ok-os-job-change-team/kaito-twitter-clone-bootcamp/pull/54/commits/0c600ce186554e65e7b361e1991035511da46ba5